scripts/extract_strings.sh

```diff
@@ -125,4 +120,17 @@ cp -r $L10N_DIR/locale/fy_NL/* $L10N_DIR/locale/fy
 sed -i'' -e 's/Language: fy_NL/Language: fy/g' "$L10N_DIR/locale/fy/LC_MESSAGES/client.po"
 sed -i'' -e 's/Language: fy_NL/Language: fy/g' "$L10N_DIR/locale/fy/LC_MESSAGES/server.po"
 
+# Fluent extraction
+cp $PAYMENTS_DIR/public/locales/en-US/*.ftl $L10N_DIR/locale/templates
+cp $SETTINGS_DIR/public/locales/en-US/*.ftl $L10N_DIR/locale/templates
+cp $MAILER_DIR/public/locales/en/*.ftl $L10N_DIR/locale/templates
+
+# Pontoon will read from the "templates" directory but we must copy the FTL files
+# into "en" to ensure users with English will always see good fallback text.
+# We can remove the "templates" directory once we are off of gettext entirely -
+# then, Pontoon will read from the "en" directory.
+cp $PAYMENTS_DIR/public/locales/en-US/*.ftl $L10N_DIR/locale/en
+cp $SETTINGS_DIR/public/locales/en-US/*.ftl $L10N_DIR/locale/en
+cp $MAILER_DIR/public/locales/en/*.ftl $L10N_DIR/locale/en
```

Comment:
Is there a reason not to use `cp $L10N_DIR/locale/templates/*.ftl $L10N_DIR/locale/en` vs 3 commands?

The advantage is that this line will remain the same, no matter if one of the packages change locale code later.

Reply:
I did that to see explicitly what's copied over but you're right, that is better. 😁 Pushed the update.